Fix error labels when missing LTC configuration

PWKad opened the pull request:
Fix #915 by capturing the current daemonName and using that in
error messages.

cfromknecht commented on config.go:
```diff
@@ -805,20 +805,27 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 		if conf.RPCUser != "" || conf.RPCPass != "" {
 			return fmt.Errorf("please set both or neither of " +
-				"btcd.rpcuser and btcd.rpcpass")
+				daemonName + ".rpcuser and " + daemonName + ".rpcpass")
```
Since we’re already using `Errorf`, this can use formatting directives instead of string concatenation

PWKad replied:
I'm not super-familiar with go, going to update the commit like this - 

```
                        return fmt.Errorf("please set all or none of " +
                                "%[1]v.rpcuser, %[1]v.rpcpass, " +
                                "and %[1]v.zmqpath", daemonName)
```

Does that look correct?

cfromknecht commented on config.go:
```diff
@@ -828,21 +835,27 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
-				"bitcoind.rpcuser, bitcoind.rpcpass, and " +
-				"bitcoind.zmqpath")
+				daemonName + ".rpcuser, " + daemonName +  ".rpcpass, " +
+				"and " + daemonName + ".zmqpath")
```
Same comment about formatting instead of string concat

PWKad commented:
@cfromknecht  updated whenever you get a chance to have a look.

cfromknecht commented on config.go:
```diff
@@ -861,20 +861,27 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 			return nil
 		}
+
+		// Get the daemon name for displaying proper errors
```
nit: we try to use complete sentences in all of our comments, just a period would suffice

cfromknecht commented on config.go:
```diff
@@ -884,21 +891,27 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 			return nil
 		}
+
+		// Get the daemon name for displaying proper errors
```
nit: same as above

PWKad commented:
Ok thank for pointing that out, I must have missed that when reading over the contribution guidelines but will check them out again.

Updated the commit message and added the periods for the sentences.  Typically since this is a fork I've just been force pushing to keep one commit but that kind of messes with GitHubs code review features so if there is a better workflow just let me know.

patrick-walters-q2 commented:
Oh nice, no I think I have a pretty high knowledge of git but somehow have never seen or used that feature, will definitely keep that in mind.  Thanks for sharing.
